metabase/metabase
# allow reordering dashboard tabs

Part of epic https://github.com/metabase/metabase/issues/29502

### Description

This PR introduces new functionality to change the order of tabs on dashboards by dragging and dropping.

### How to verify

1. Create new dashboard
2. Edit -> add tabs
3. Should be able to drag tabs around while editing
4. Save -> tabs should still be in new order

### Demo

https://github.com/metabase/metabase/assets/37751258/36b651a9-d079-443c-a9fd-19f84546be8c

### Checklist



- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/30833)
